include/drops/drops.hpp

```diff
@@ -82,54 +87,79 @@ class [[eosio::contract("drops")]] drops : public contract
       bool            enabled        = true;
    };
 
-   struct [[eosio::table("unbind")]] unbind_row
+   /**
+    * ## TABLE `balances`
+    *
+    * ### params
+    *
+    * - `{name} owner` - (primary key) owner account
+    * - `{int64_t} drops` - total amount of drops owned
+    * - `{int64_t} ram_bytes` - total amount of RAM bytes available by the owner
+    *
+    * ### example
+    *
+    * ```json
+    * {
+    *   "owner": "test.gm",
+    *   "drops": 69,
+    *   "ram_bytes": 2048
+    * }
+    * ```
+    */
+   struct [[eosio::table("balances")]] balances_row
+   {
+      name    owner;
+      int64_t drops;
+      int64_t ram_bytes;
+
+      uint64_t primary_key() const { return owner.value; }
+   };
+
+   /**
+    * ## TABLE `stat`
+    *
+    * ### params
+    *
+    * - `{int64_t} drops` - total supply of drops
+    * - `{int64_t} ram_bytes` - total available RAM bytes held by the contract
+    *
+    * ### example
+    *
+    * ```json
+    * {
+    *   "drops": 88888,
+    *   "ram_bytes": 2048
+    * }
+    * ```
+    */
+   struct [[eosio::table("stat")]] stat_row
    {
-      name             owner;
-      vector<uint64_t> drops_ids;
-      uint64_t         primary_key() const { return owner.value; }
+      int64_t drops;
+      int64_t ram_bytes;
    };
 
    typedef eosio::multi_index<
       "drop"_n,
       drop_row,
       eosio::indexed_by<"owner"_n, eosio::const_mem_fun<drop_row, uint128_t, &drop_row::by_owner>>>
-                                                      drop_table;
-   typedef eosio::singleton<"state"_n, state_row>     state_table;
-   typedef eosio::multi_index<"unbind"_n, unbind_row> unbind_table;
-
-   /*
-
-    Return value structs
-
-   */
-
-   struct generate_return_value
-   {
-      uint32_t drops;
-      asset    cost;
-      asset    refund;
-      uint64_t total_drops;
-   };
+                                                          drop_table;
+   typedef eosio::singleton<"state"_n, state_row>         state_table;
+   typedef eosio::multi_index<"balances"_n, balances_row> balances_table;
+   typedef eosio::singleton<"stat"_n, stat_row>           stat_table;
 
+   // @return
    struct destroy_return_value
    {
-      uint64_t ram_sold;
-      asset    redeemed;
-      uint64_t ram_reclaimed;
-   };
-
-   struct bind_return_value
-   {
-      uint64_t ram_sold;
-      asset    redeemed;
+      int64_t unbound_destroyed;
+      int64_t bytes_reclaimed;
```

Comment:
I don't believe these action return structs follow the `Name` schema, there's no limitations

The data is stored as Hex data, pretty sure the user can override these definitions and it would still work, as long as the data types is in the same order `[int64_t, int64_t]`